wp-content/themes/bpr2018/resources/templates/layout/head.tpl.php

```diff
@@ -186,3 +187,6 @@ class="logo"
           })
         });
       </script>
+
+<a id="top-of-content" name="top-of-content"></a>
```

Comment:
I would recommend making this into a `div` element (but keep the id and name). Screen readers may get confused about the empty `a` element.